src/main/java/no/unit/nva/database/DatabaseServiceImpl.java

```diff
@@ -47,23 +46,22 @@ public DatabaseServiceImpl(DynamoDBMapper mapper) {
     @Override
     public Optional<UserDto> getUser(String username) throws InvalidUserException {
         UserDb searchObject = UserDto.newBuilder().withUsername(username).build().toUserDb();
-        DynamoDBQueryExpression<UserDb> query = createGetUserQuery(searchObject);
-        List<UserDb> result = mapper.query(UserDb.class, query);
+        DynamoDBQueryExpression<UserDb> searchUserByUsername = createGetUserQuery(searchObject);
+        List<UserDb> userSearchResult = mapper.query(UserDb.class, searchUserByUsername);
 
-        UserDto user = result
+        UserDto user = userSearchResult
             .stream()
             .map(attempt(UserDto::fromUserDb))
-            .map(effort -> effort.orElseThrow(this::unexpectedException))
+            .map(attempt -> attempt.orElseThrow(this::unexpectedException))
```

Comment:
I didn't say previously, but the "try" framework works really well — it is easy to use and makes the code very readable